examples/nixnet/ethernet-frame-stream-input-reader.py

```diff
@@ -36,8 +36,7 @@
 
 CHOOSE_MONITOR_OR_ENDPOINT_TEXT = (
     "\nPress 'm' followed by enter to configure the input stream session to use the monitor path"
-    " to monitor all network traffic else just the enter key to use the endpoint path which filters"
-    " traffic based on VLAN ID and priority. :"
+    " to monitor all network traffic. :"
```

Comment:
You previous message was fine, it just needed rewording for clarity.

> "\nPress 'm' followed by enter to configure the input stream session to use the monitor path"
    " to monitor all network traffic else press enter key to use the endpoint path which filters"
    " traffic based on VLAN ID and priority. :"
    
The change is in `press enter key`. I assume just pressing `enter` would configure it in filtered mode.


> Press 'm' followed by enter

Is this the behavior? Do you user need to press just `m` or `m + enter` to configure as monitor.

Reply:
We need to press 'm' followed by enter because otherwise reading just a character will require importing getch and that anyways we ain't doing.

Reply:
Sure, that makes sense. And for non-monitor mode just `enter` works, right?

Reply:
No, for the else case it doesn't work with just `enter` and that's why i changed the comment.

Reply:
Okay. In that case the second half of comment should read
> else press any other key followed by enter to use the endpoint path which filters